docs/content/Buttons.md

```diff
@@ -11,20 +11,22 @@ To create a button group, wrap `Button` elements in the `ButtonGroup` element. `
 ## Default examples
 
 ```jsx live
-<Button>Button</Button>
-<ButtonDanger>Button Danger</ButtonDanger>
-<ButtonOutline>Button Outline</ButtonOutline>
-<ButtonPrimary>Button Primary</ButtonPrimary>
-<ButtonInvisible>Button Invisible</ButtonInvisible>
-<ButtonClose onClick={() => window.alert('button clicked')}/>
-
-<ButtonGroup display='block' my={2}>
+<>
   <Button>Button</Button>
-  <Button>Button</Button>
-  <Button>Button</Button>
-</ButtonGroup>
-
-<ButtonTableList>Button Table List </ButtonTableList>
+  <ButtonDanger>Button Danger</ButtonDanger>
+  <ButtonOutline>Button Outline</ButtonOutline>
+  <ButtonPrimary>Button Primary</ButtonPrimary>
+  <ButtonInvisible>Button Invisible</ButtonInvisible>
+  <ButtonClose onClick={() => window.alert('button clicked')}/>
+
+  <ButtonGroup display='block' my={2}>
+    <Button>Button</Button>
+    <Button>Button</Button>
+    <Button>Button</Button>
+  </ButtonGroup>
+
+  <ButtonTableList>Button Table List </ButtonTableList>
+</>
```

Comment:
Was rendering these components outside a fragment causing a lint error?

Reply:
It's not a lint error itself, eslint gave a parse error and failed because returning more than one element is not valid ```jsx``` code.